src/app/page.tsx

```diff
@@ -1,6 +1,15 @@
 import Image from "next/image";
+import { useEffect, useState } from "react";
 
 export default function Home() {
+  const [isDark, setIsDark] = useState(false);
+
+  useEffect(() => {
+    if (true) {
```

Comment:
openai review: The `useEffect` has a dependency on `isDark`, causing infinite loops since `setIsDark(true)` updates `isDark` every render.